python/samples/concepts/memory/azure_ai_search_hotel_samples/README.md

```diff
@@ -0,0 +1,72 @@
+## Azure AI Search with Hotel Sample Data
+
+This guide walks you through setting up your Azure AI Search Service with the correct index, data source, and indexer to run the hotel sample.
+
+### Setting Up the Azure AI Search Service
+
+1. **Import the Sample Data**  
+   - Navigate to the **Search Service Overview** page and click **Import Data**.
+   - From the dropdown, select **Samples**, then choose **hotels-sample**.
+   - Click **Next: Add Cognitive Skills (Optional)**.
+
+2. **Skip the Cognitive Skills Page**  
+   - No changes are needed here. Click **Next** to proceed.
+
+3. **Configure the Index Fields**  
+   - The Python sample uses **snake_case** field names. Update the default field names accordingly.
+   - Since `HotelId` is the primary key, you cannot rename it directly. Instead, create a new field:
+     - Click **+ Add Field** and name it `hotel_id`.
+     - Enable **Retrievable**, **Filterable**, **Facetable**, and **Searchable** options.
+   - Rename other fields to snake case:
+     - `HotelName` → `hotel_name`
+       - There may be a current issue with index config that has trouble mapping the `HotelName` -> `hotel_name`, so as to not hit issues
+         deselect `retrievable` for `hotel_name`. It should still be `searchable`.
+     - Use the dropdown to rename complex fields like `Address` -> `address` and `Rooms` -> `rooms` with their sub-fields renamed.
+   - Add two new vector fields:
+     - `description_vector`
+     - `description_fr_vector`
+   - Configure these fields as:
+     - **Type**: `Collection(Edm.Single)` (for vector fields)
+     - **Retrievable**: Enabled (default setting)
+     - Click the **three dots (...)** on the right, then **Configure vector field**:
+       - Set **Dimensions** to `1536`.
+       - If no vector search profiles exist, click **Create**.
+       - Under **Algorithms**, click **Create** to set up a vector algorithm (default values are fine).
+       - If no vectorizer exists, create one:
```

Comment:
@eavanvalkenburg I couldn't get the sample to work properly until I defined the vectorizer in my index, even though I am using a different vectorizer from the concept sample. Curious about your experience here.

Reply:
might have been a change in the portal, I set this up so long ago that I don't remember :)